projects/storefrontapp-e2e-cypress/cypress/helpers/customer-ticketing/customer-ticketing.ts

```diff
@@ -149,3 +149,19 @@ export function verifyTicketDoesNotExist(ticketDetails: TestTicketDetails) {
 export function visitApparelUKTicketListingPage(){
   visitPage('apparel-uk-spa/en/GBP/my-account/support-tickets', 'apparelTicketListingPage');
 }
+
+export function verifyCreateTicketPopupIsClosed(){
+  cy.get('cx-customer-ticketing-create-dialog').should('not.exist');
+}
+
+export function verifyFieldInputDoesntExceedLimit(){
+  cy.get('cx-customer-ticketing-create-dialog').within(() => {
+    cy.get('textarea').first().invoke('text').then((text) => {
```

Comment:
There might be a better way to identify the subject input. Maybe using the Aria label

Reply:
Seems like there is no aria labels on the create ticket dialog box

Reply:
What about using a different approach:
```suggestion
    const SUBJECT_LABEL = 'Subject';
    cy.get('label').contains(SUBJECT_LABEL).within(() => {
        cy.get('textarea').invoke('text').then(subject => {
            expect(subject.length).to.be.lte(TICKET_SUBJECT_MAX_LENGTH);
        });
    });
    
```
Other option could be:
```suggestion
   const SUBJECT_CONTROL = 'subject';
   cy.get(`textarea[formcontrolname="${SUBJECT_CONTROL}"]`).invoke('text').then(subject => {
        expect(subject.length).to.be.lte(TICKET_SUBJECT_MAX_LENGTH);
    });
```